README-CN.md

```diff
@@ -0,0 +1,163 @@
+# K8SMeetup 翻译流程与翻译校稿规范
+
+time：2018-07-11 author：xiaolong@caicloud.io
+
+由于 `website` 上游仓库从在 2018年7月9日，从 `Jeklly` 成功迁移至 `hugo`，且提供了完善的中文化支持，目前官网 `kubernetes.io` 已经可以轻松切换中文站点。因此原有的 [中文翻译仓库](https://github.com/kubernetes/kubernetes-docs-zh) 不在维护，继续冻结中。往期的翻译内容仍然可以[这里](https://k8smeetup.github.io)访问，对应往期的翻译者贡献仍然可以[这里](https://k8smeetup.github.io/chart/) 查看。
```

Comment:
i think this is misunderstood - from the email that Zach sent on June 20, there's a line saying "We changed the name of the Chinese translation repository to k/docs-kubernetes-zh. GitHub automatically aliases the new name, so contributors should experience no problems." If I understand correctly, it's still the same repo but with a different alias (name) now.